samples/web/content/apprtc/chrome-app-build/fetch-fonts.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import sys
+import re
+import urllib2
+from urlparse import urlparse
+from os.path import basename
+
+parser = argparse.ArgumentParser(description='Parse css file and retrieve all referenced font files')
```

Comment:
Sure, @juberti suggested I break up the PR into smaller pieces below, so I can do the font pieces separately as well.